onecodex/viz/_bargraph.py

```diff
@@ -0,0 +1,124 @@
+from onecodex.exceptions import OneCodexException
+import altair as alt
```

Comment:
Let's try to keep the imports in two groups: xeno-modules at the top, then those from within `onecodex`, with an empty line between the two groups. And put them in alphabetical order. I might have messed this up elsewhere myself.